docker/.env

```
@@ -260,3 +260,62 @@ UV_GUARDRAILS_DB_NAME=guardrails
 TRAEFIK_HTTP_PORT=__TRAEFIK_HTTP_PORT__
 TRAEFIK_HTTPS_PORT=__TRAEFIK_HTTPS_PORT__
 TRAEFIK_DASHBOARD_PORT=__TRAEFIK_DASHBOARD_PORT__
+
+## OpenBao PKI
+AM_CERTS_OPENBAO_HOST=http://certs-openbao:8200
+AM_CERTS_OPENBAO_APP_ROLE=absmach
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AM_CERTS_OPENBAO_APP_ROLE key should go before the AM_CERTS_OPENBAO_HOST key

(UnorderedKey)